src/features/presence/__tests__/__snapshots__/Presence-test.js.snap

```diff
@@ -46,7 +46,7 @@ exports[`features/presence/Presence render() Tests for presence autofly - GROWTH
                 id="boxui.presence.timeSinceLastPreviewed"
                 values={
                   Object {
-                    "timeAgo": <UNDEFINED
+                    "timeAgo": <FormattedRelative
```

Comment:
adding the update to the mock react intl file prevents this from rending invariant errors in the test now, and allows for more meaningful snapshots in future components (more to fix in the mocks tho).